benchmark/parallel/egonets.jl

```diff
@@ -1,62 +1,57 @@
 using Graphs
 using BenchmarkTools
-@show Threads.nthreads()
-
-@benchgroup "parallel" begin
-    @benchgroup "egonet" begin
-        function vertex_function(g::Graph, i::Int)
-            a = 0
-            for u in neighbors(g, i)
-                a += degree(g, u)
-            end
-            return a
-        end
-
-        function twohop(g::Graph, i::Int)
-            a = 0
-            for u in neighbors(g, i)
-                for v in neighbors(g, u)
-                    a += degree(g, v)
-                end
-            end
-            return a
-        end
 
-        function mapvertices(f, g::Graph)
-            n = nv(g)
-            a = zeros(Int, n)
-            Threads.@threads for i in 1:n
-                a[i] = f(g, i)
-            end
-            return a
-        end
+SUITE["parallel"] = BenchmarkGroup([], "egonet" => BenchmarkGroup([]))
```

Comment:
this is no longer necessary from BenchmarkTools v1.4 cause nested dict keys are created on the fly

Reply:
yes, but let's do that kind of content modifications later :)